### DISCUSSION TITLE
Install on Heroku

### DISCUSSION THREAD

**Discussion by ghost** [dani-garcia/vaultwarden#1472] [General]
Hello! Can I install it on Heroku?

**Comment by dani-garcia** [comment]
We haven't had any users try that before, but as long as you can deploy either the docker image or build the rust code itself in the heroku machine it should work. I'm afraid I can't help you with the details, as I've never used heroku.

**Comment by mqus** [comment]
If you want to help other users with installing it on heroku and think it will help, you are welcome to create a wiki article. If you encounter any problems, try posting on the forum. (Both are linked to in the readme file of this project)

**Comment by fxzxmic** [comment]
![image](https://user-images.githubusercontent.com/54622331/96373841-a163a100-11a1-11eb-9b7d-97d6ed21f77b.png)
I don't know what went wrong.

**Comment by davidjameshowell** [comment]
I know this is closed, but I wanted to link to an example of Heroku deploy I had been working on.

https://github.com/davidjameshowell/bitwarden_rs_heroku

It's a really simple script and I don't begin to call my Bash scripting good by any means, but this sets up a Bitwarden_rs instance ready to use.

**Comment by std2main** [comment]
> ![image](https://user-images.githubusercontent.com/54622331/96373841-a163a100-11a1-11eb-9b7d-97d6ed21f77b.png)
> I don't know what went wrong.

The reason is that ROCKET_PORT is 80 by default.
It need to be adjusted to $PORT that provided by heroku.

Here is my simple work that deploy bitwarden_rs on heroku.

https://github.com/std2main/bitwardenrs_heroku
